### PR TITLE
Install hugo on windows11

### PR DESCRIPTION
**Pull request by TheDoubleH**
Instruction for manually install hugo

### CODE REVIEW

**Review by JeremyVyska — APPROVED** [review]

**Comment by TheDoubleH** [comment]
Thanks for the proof reading, @JeremyVyska .

**Comment by waldo1001** [comment]
I have the feeling this procedure is more difficult than the script @ajkauffmann came up with.  MOreover, the current description didn't work for me.

Here's AJ's script:
```powershell
# SYSTEM PREPARATION #
Set-ExecutionPolicy Bypass -Scope Process -Force
[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072
Invoke-Expression ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))
choco install -y nodejs
choco install -y hugo-extended

# RUN INSIDE REPO #
git submodule update --init --recursive --depth 1
npm install
```

Shall we move to that one?

**Review by waldo1001 — CHANGES_REQUESTED** [review]

File: content/docs/Contributing/InstallHugo/ManuallyOnWindows11/index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+![image](EnvironmentVariables.png)
+
+Now press `New` and add the `C:\Hugo\Bin` to the path. Press `OK` and `OK` to save the new `Path`
+
+![image](EnvironmentVariablesPath.png)
+
+Once complete. You should now be able to preview the Hugo site on <http://localhost:1313> by opening a command promt, and open the `C:\Hugo\Sites\alguidelines` folder and execute `Hugo Serve`
+
+![image](HugoServe.png)
```
[the remark this answers is not in the document]
Comment (TheDoubleH) [reply content/docs/Contributing/InstallHugo/ManuallyOnWindows11/index.md:57]: Did You verify that You did in fact have the 'Extended' version of Hugo installed?

**Comment by TheDoubleH** [comment]
> I have the feeling this procedure is more difficult than the script @ajkauffmann came up with. MOreover, the current description didn't work for me.
> 
> Here's AJ's script:
> 
> ```powershell
> # SYSTEM PREPARATION #
> Set-ExecutionPolicy Bypass -Scope Process -Force
> [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072
> Invoke-Expression ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))
> choco install -y nodejs
> choco install -y hugo-extended
> 
> # RUN INSIDE REPO #
> git submodule update --init --recursive --depth 1
> npm install
> ```
> 
> Shall we move to that one?

I think we can ADD that one. I purposefully created the Installation MD file structure, so that different methods could be created.

File: content/docs/Contributing/InstallHugo/ManuallyOnWindows11/index.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+├── Bin            # Binaries
+└┬─ Sites          # Hugo Site Repositories
+ └── alguidelines  # Hugo Source for alguidelines.dev
+```
+
+{{% /alert %}}
+
+![image](SelectExtended.png)
+
+After downloading the .zip file, extract the zip-file to `c:\Hugo\Bin`
```
[the remark this answers is not in the document]
Comment (TheDoubleH) [reply content/docs/Contributing/InstallHugo/ManuallyOnWindows11/index.md:29]: You extract the zip-file directly

File: content/docs/Contributing/InstallHugo/ManuallyOnWindows11/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (TheDoubleH) [reply content/docs/Contributing/InstallHugo/ManuallyOnWindows11/index.md:57]: I have to try on a fresh machine - I might have installed node.js as well

**Comment by TheDoubleH** [comment]
Added page describing the install via powershell

File: content/docs/Contributing/InstallHugo/ManuallyOnWindows11/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (TheDoubleH) [reply content/docs/Contributing/InstallHugo/ManuallyOnWindows11/index.md:57]: image

**Review by waldo1001 — APPROVED** [review]

**Review by ajkauffmann — APPROVED** [review]

**Pull request merged**